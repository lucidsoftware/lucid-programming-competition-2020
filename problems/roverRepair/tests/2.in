15
50
19
72
12
23
54
76
9
14
17
empty
empty
67
empty
empty
