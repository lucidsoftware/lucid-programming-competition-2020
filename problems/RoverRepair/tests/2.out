Swap nodes 17 and 19
